src/libs/ajax/Billing.js

```diff
@@ -26,10 +26,11 @@ export const Billing = signal => ({
     )
   },
 
-  createAzureProject: async (projectName, tenantId, subscriptionId, managedResourceGroupId) => {
+  createAzureProject: async (projectName, tenantId, subscriptionId, managedResourceGroupId, members) => {
+    // members: an array of {email: string, role: string}
     return await fetchRawls('billing/v2',
       _.mergeAll([authOpts(), jsonBody(
-        { projectName, managedAppCoordinates: { tenantId, subscriptionId, managedResourceGroupId } }
+        { projectName, managedAppCoordinates: { tenantId, subscriptionId, managedResourceGroupId }, members }
```

Comment:
I was under the impression that we want users to be able to invite others to Terra while creating a billing project, in which case we'd need to set `inviteUsersNotFound` to true or Rawls will throw if a user is invited during billing project creation

Reply:
Yes, we should set inviteUsersNotFound to true. That will likely also take care of the 409 issue.